jbpm-wb-executor-service/jbpm-wb-executor-service-api/src/main/java/org/jbpm/workbench/es/model/RequestSummary.java

```diff
@@ -123,6 +125,10 @@ public Integer getRetries() {
         return retries;
     }
 
+    public void setRetries(Integer retries) {
+        this.retries = retries;
+    }
+
```

Comment:
If this is the setter we want to have for ```retries```, I guess we could then delete the existing ```setRetries(int retries)```.

Reply:
+1